main/modes/mode_flight.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * mode_flight.h
+ *
+ *  Created on: Sept 15th, 2020
+ *      Author: <>< CNLohr
+ */
+
+#ifndef MODES_MODE_FLIGHT_H_
+#define MODES_MODE_FLIGHT_H_
+
+extern swadgeMode modeFlight;
+
+#define NUM_FLIGHTSIM_TOP_SCORES 4
+#define FLIGHT_HIGH_SCORE_NAME_LEN 4
+
+typedef struct __attribute__((aligned(4)))
```

Comment:
I need to implement those blob functions for NVS...

Reply:
See https://github.com/AEFeinstein/Super-2023-Swadge-FW/issues/89

Reply:
Unless you want to copy `readNvs32()` and `writeNvs32()`, then replace the calls to `nvs_set_i32()` and `nvs_get_i32()` with `nvs_set_blob()` and `nvs_get_blob()` respectively.

Reply:
#89 is done